### ISSUE TITLE
cpu name is not displayed

### ISSUE THREAD

**Issue by promeneur** [sainsaar/gkrellm2-cpupower#4]
opensuse 42.3

only profile and frequency is displayed not core name like Cpu0, Cpu1 etc.

**Comment by tomaszg7** [comment]
I don't think it was supposed to. It would use much more vertical space. It's probably not very hard to add this feature.

**Comment by promeneur** [comment]
"It would use much more vertical space."

you are right. 

Is it possible to display as this:
Cpu 0 "profile" "frequency"
Cpu 1 "profile" "frequency"
...
Cpu n "profile" "frequency"

and to not display the little cursor  of frequency which is not usefull in the case user does not get the right to manage frequency. This is the general case with opensuse.

Thanks

**Comment by promeneur** [comment]
another solution:

insert "profile" "frequency" in same window of gkrellm builtins CPU
